x/bridge/keeper/assets_locked.go

```diff
@@ -96,7 +96,22 @@ func (k Keeper) AcceptAssetsLocked(
 			return fmt.Errorf("failed to parse recipient address: %w", err)
 		}
 
+		// we need to ensure that the funds are not sent
+		// to a blocked address first, in case they are blocked,
+		// then no minting should happen, the funds will
+		// be locked in the bridge.
```

Comment:
Let's cleanup this comment:
```
		// We need to ensure that the funds are not sent to a blocked address first. 
        // In case the address is blocked, no minting should happen, and the funds should
		// remain locked in the bridge.
```

Reply:
addressed in 34c05ba4c2ca492883883a4a4489e646382cdea4